Bookish.Web/Views/Home/Catalogue.cshtml

```diff
@@ -53,5 +53,15 @@ else
                 </tr>
             }
         </table>
+        <nav>
+            <ul class="pagination">
```

Comment:
One more thing, if all the books fit on one page then I'd expect the pagination items to be hidden 🙂